docs/documentation/custom_modules/add_plugin.rst

```diff
@@ -60,10 +150,29 @@ Let's assume your :code:`pyproject.toml` file is configured so that your project
     name = "ST_drive_models"
     version = "1.0.0"
 
+    # Tells location of sources
+    packages = [
+        { include = "star_trek", from = "src" },
+    ]
+
+    ...
+
+    # Specify that Poetry is used for building the package
+    [build-system]
+    requires = ["poetry-core>=1.0.0"]
+    build-backend = "poetry.core.masonry.api"
+
+    ...
+
+    # Plugin declaration
+    [tool.poetry.plugins."fastoad.plugins"]
+    "ST_plugin" = "star_trek.drives"
     ...
 
-It will create a :code:`dist` folder with two files: :code:`ST_drive_models-1.0.0.tar.gz`
-and :code:`ST_drive_models-1.0.0-py3-none-any.whl` (or something like this).
+The command :code:`poetry build` will create a :code:`dist` folder with two files:
+
+:code:`ST_drive_models-1.0.0.tar.gz` and :code:`ST_drive_models-1.0.0-py3-none-any.whl`
+(or something like this).
 
 You may then have sent any of those two files to another user, who may then install your models
 using :code:`pip` with:
```

Comment:
In the section "Publishing" I would underline that to publish on PyPi the package name will be the one provided in the field `name` in the `pyproject.toml`. Therefore, the publisher shall check that the chosen package name is not already taken on PyPi.

Reply:
Agreed. Done in a64d130b46db8f1102f014668d822f0b528c5921.